lib/index.js

```diff
@@ -11,29 +11,26 @@ module.exports = function nuxtMarkdownit (options) {
   }
 
   const version = this.nuxt.constructor.version
-  const [major, minor, patch] = version.split('.')
+  const [major] = version.split('.')
 
   this.extendBuild(config => {
     if (major === '1') {
```

Comment:
@pi0 Should we keep this major check on nuxt version since Nuxt 2 has been released on 20 Sep 2018 ?

Reply:
You can remove it @Geminii . Anyway need to bump major version of module (since a legacy project can still use nuxt 1 and this package correct way is to bump major even if nuxt 2 released 2 years ago some projects may still use nuxt 1 i personally have a 0.x deployment :)))))